packages/gatsby-source-wordpress/src/fetch.js

```diff
@@ -453,6 +466,11 @@ function getValidRoutes({
             break
         }
         validRoutes.push({ url: route._links.self, type: validType })
+      } else {
+        if (_verbose)
+          console.log(
+            colorized.out(`Invalid route.`, colorized.color.Font.FgRed)
+          )
```

Comment:
Maybe we could keep track why this route is excluded and print out that either this route wasn't whitelisted or it was blacklisted?